frontend/src/cards/MapCard.tsx

```diff
@@ -178,9 +178,12 @@ function MapCardWithKey(props: MapCardProps) {
   let subtitle = createSubtitle({ currentBreakdown, activeBreakdownFilter });
   filename = `${filename} ${subtitle ? `for ${subtitle}` : ""}`;
 
-  if (HIV_DETERMINANTS.includes(metricConfig.metricId)) {
-    if (subtitle === "") subtitle = "Ages 13+";
-    else if (currentBreakdown !== "age") subtitle += ", Ages 13+";
+  if (isPopulationSubset) {
+    let ageTitle = metricConfig.metricId.includes("prep")
```

Comment:
Should we do an exact string match here?

Reply:
yes, and probably should ideally make it a function or extendable map since we have other subset populations we should be adding as well (like voting being 18+ citizens, etc)